bundles/org.openhab.binding.resol/src/main/java/org/openhab/binding/resol/handler/ResolThingHandler.java

```diff
@@ -249,11 +249,12 @@ protected void packetReceived(Specification spec, Language lang, Packet packet)
                                     this.updateState(channelId, q);
                                 } else {
                                     try {
-                                        QuantityType<?> q = new QuantityType<>(str);
+                                        QuantityType<?> q = new QuantityType<>(str, Locale
+                                                .getDefault()); /* vbus library returns the value in default locale */
                                         this.updateState(channelId, q);
                                     } catch (IllegalArgumentException e) {
                                         logger.debug("unit of '{}' unknown in openHAB", str);
-                                        QuantityType<?> q = new QuantityType<>(dd.toString());
+                                        QuantityType<?> q = new QuantityType<>(dd.toString(), Locale.getDefault());
```

Comment:
`dd` is a `Double`. I don't think you need to pass the locale there.

You are creating a `QuantityType` without any unit ?? That looks strange to me.

Reply:
Ok, you are passing a string without any unit and a locale. I don't know what will be the unit considered by the core framework in that case.

Reply:
I would have imagined to rather use in that case the constructor with a Number and a unit.

https://github.com/openhab/openhab-core/blob/4f35d474fc63f487b10042d05f01ea524fe312e8/bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/QuantityType.java#L164

Reply:
I would prefer that yes, but this code is only executed if the resol-vbus library returns a unit that is not known by openHAB. So I cannot convert it to a reasonable unit. I actually don't remember whether it commonly happens or whether it is only for a case that could theoretically. In this PR I only added the locale to cover the localized decimal point correctly - which was a change in the lib between 0.7 and 0.10.

Reply:
ah no, dd is the Double, so after the toString the QuantityType should not need special handling of the decimal point. You're totally right, adding the locale here is useless and even wrong if the default local uses , as decimal separator.

Would 
`QuantityType<?> q = new QuantityType<>(dd, AbstractUnit.ONE);`
be better?